app/src/main/java/uk/ac/ebi/atlas/home/JsonExperimentsSummaryController.java

```diff
@@ -47,6 +47,11 @@ private ImmutableList<CardModel> featuredExperimentsCards() {
                         getExperimentsSummaryImageUrl("hca"),
                         Pair.of(Optional.of("Human Cell Atlas"), Optional.of(getCustomUrl("/hca-landing-page.html"))),
                         ImmutableList.of()),
+                CardModel.create(
+                        IMAGE,
+                        getExperimentsSummaryImageUrl("flycellatlas"),
```

Comment:
please format `flycellatlas` to `fly-cell-atlas` in accordance to other card models.